packages/react-components/src/components/Loader/Loader.tsx

```diff
@@ -9,7 +9,7 @@ const baseClass = 'loader';
 const spinnerClass = `${baseClass}__spinner`;
 
 export interface LoaderProps extends React.HTMLAttributes<HTMLDivElement> {
-  size?: 'small' | 'medium' | 'large';
+  size?: 'small' | 'medium' | 'large' | 'cover';
```

Comment:
For me we should introduce `fullWidth` prop instead of adding new `size`.
Size prop is generic and used in other components, why would we be able to choose more options for prop size here?

Reply:
Hmm, in <Loader /> component, `size` defaulted to `medium`. I wanted to provide another size prop, because it felt natural for me. So you're saying that adding another prop `fullWidth` is more benefitial instead of having another `size`?

Reply:
This is my subjective opinion, but yes - we shouldn't be afraid of adding new props. For me, the size should be constant not that one component has 4 size options and the other has 3.
When looking at such MUI or Chakra UI they have clearly defined sizes of their components and some additional behaviors come in the form of new props.

If cover is going to take 100% width and 100% height then in this case maybe I would change my proposal for prop name to `cover` instead of `fullWidth` or any other name but I would not extend  size prop

Reply:
I also have a bit of mixed feelings about this one. Do we have a valid use case for Design System users? If we would like to include the ability for components to expand to full width we would need to support it for most of them. Personally, I feel it might not be that necessary. If we want it to handle the specific case within the DS itself I would simply implement it using CSS within the `Switch` component.

Reply:
Okay so you are saying to abandon `cover` size for `Loader` component. And instead we should provide our own loading Icon to the `Switch` component and make animation there and not use <Loader /> inside `Switch` at all?

Reply:
As discussed F2F I'd suggest using `Loader` component with custom styling within `Switch` for the loading state.

Reply:
Fixed - [1d02631](https://github.com/livechat/design-system/pull/372/commits/1d026314ab2284f8ecfd71d1988e81e0841a0383)